components/BottomNavBar.tsx

```diff
@@ -0,0 +1,66 @@
+import Link from 'next/link';
+import DashboardIcon from './icons/DashboardIcon';
+import SearchIcon from './icons/SearchIcon';
+import SiteListIcon from './icons/SiteListIcon';
+import { useRouter } from 'next/router';
+import useSWR, { Fetcher } from 'swr';
+import { SiteListProps } from '~/lib/types';
+
+const icons = [
+  {
+    title: 'Dashboard',
+    icon: DashboardIcon,
+    link: '/dashboard',
+  },
+  {
+    title: 'My Sites',
+    icon: SiteListIcon,
+    link: '/sites',
+  },
+  {
+    title: 'More Info',
+    icon: SearchIcon,
+    link: '/more-info',
+  },
+];
+
+const BottomNavBar = () => {
+  const { asPath } = useRouter();
+  const fetcher: Fetcher<SiteListProps> = async (url: string) => {
+    return await fetch(url).then((res) => res.json());
+  };
```

Comment:
No need to define a fetcher than turns the request into JSON anymore since we've got one defined inside the `_app.tsx` as the default. Isn't that cool.
```suggestion

```